testplan/web_ui/testing/src/Toolbar/FilterBox.js

```diff
@@ -2,24 +2,28 @@ import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import {StyleSheet, css} from 'aphrodite';
 
-import {COLUMN_WIDTH} from "../Common/defaults";
+//import {COLUMN_WIDTH} from "../Common/defaults";
 
 /**
  * Filter box, enter filter expressions to filter entries from the Nav
  * component.
  */
 class FilterBox extends Component {
-  render() {return (
-    <div
-      className={css(styles.searchBox)}
-      style={{width: `${COLUMN_WIDTH}em`}}>
-      <input
-        className={css(styles.searchBoxInput)}
-        type="text"
-        placeholder="&#xf002; Filter"
-        onKeyUp={this.props.handleNavFilter} />
-    </div>
-  );
+  render() {
+    console.log('===Paint filterbox===');
```

Comment:
Do we need to keep these temporary logs? I think they can be removed before this is merged.

Reply:
removed